lib/diazo/rules.py

```diff
@@ -121,7 +129,10 @@ def expand_themes(rules_doc, parser=None, absolute_prefix=None,
         if not read_network and url[:6] in ('ftp://', 'http:/', 'https:'):
             raise ValueError("Supplied theme '%s', "
                              "but network access denied." % url)
-        theme_doc = etree.parse(url, parser=parser)
+        if url.startswith('https://'):
+            theme_doc = html.parse(urlopen(url))
+        else:
+            theme_doc = etree.parse(url, parser=parser)
```

Comment:
Well would be simpler but, that would break context at the moment, due to fix binding to an html parser instead of an generic parser pass in by parameter. I will look more in detail to fix it.